runtime/kusama/constants/Cargo.toml

```diff
@@ -10,10 +10,14 @@ smallvec = "1.8.0"
 frame-support = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 primitives = { package = "polkadot-primitives", path = "../../../primitives", default-features = false }
 runtime-common = { package = "polkadot-runtime-common", path = "../../common", default-features = false }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
+sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "master" }
+sp-weights = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "master" }
+sp-core = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "master" }
```

Comment:
Why these changes?

Reply:
Needed to make it compile since we moved the weight stuff to `sp-weights` and `parameter_types` is now imported from `sp_core` instead of `frame_support` (dont know why). I think the change was part of the WeightV2 refactor.